entrypoint.sh

```diff
@@ -7,6 +7,8 @@ OLD_APT_DEPENDENCIES=$1
 CODECOV_TOKEN=$2
 CMAKE_ARGS=$3
 
+export DEBIAN_FRONTEND="noninteractive"
```

Comment:
Does this need to be backported into the bionic branch?

Reply:
Even though it's not causing errors there, I think it's best to. I'll do it once this PR is in.